ffs/filcold/filcold.go

```diff
@@ -86,9 +86,6 @@ func (fc *FilCold) Store(ctx context.Context, c cid.Cid, cfg ffs.FilConfig) ([]c
 	if err != nil {
 		return nil, nil, 0, fmt.Errorf("starting deals: %s", err)
 	}
-	if len(okDeals) == 0 {
-		return nil, failedStartingDeals, 0, fmt.Errorf("all proposed deals where rejected")
-	}
```

Comment:
`Store` should be biased by the result. It's the caller responsibility to consider if his use-case is a failure or not. Store just fire things and indicates an error in extraordinary conditions. Rejected deals aren't extraordinary.
The caller can just check that all the deals failed if is in its interests (i.e: the `len` of the failed slice is equal to all proposed).